### PR TITLE
Adding tests and fix for duplicate org administrators for collection …

### PR DESCRIPTION
**Pull request by GERey**
…connections. Only service level verification

### CODE REVIEW

**Comment by GERey** [comment]
https://issues.apache.org/jira/browse/USERGRID-1187

File: stack/services/src/main/java/org/apache/usergrid/management/cassandra/ManagementServiceImpl.java [hunk 1]
```diff
@@ -1608,6 +1608,12 @@ public void addAdminUserToOrganization( UserInfo user, OrganizationInfo organiza
         }
 
         EntityManager em = emf.getEntityManager( smf.getManagementAppId() );
+
+        if(em.getCollection(organization.getUuid() ,"users",Query.fromQL( "select * where uuid ="+user.getUuid() ),Level.IDS ).size() >0){
+            logger.trace( "Found value: {} already in collection",user.getName() );
```
Comment (michaelarusso) [inline stack/services/src/main/java/org/apache/usergrid/management/cassandra/ManagementServiceImpl.java]: maybe change to debug level and wrap in if statement for isDebugEnabled ?
